packages/wallet-lib/src/types/Account/methods/broadcastTransaction.js

```diff
@@ -74,14 +98,44 @@ async function broadcastTransaction(transaction, options = {}) {
   const serializedTransaction = transaction.toString();
 
   const txid = await this.transport.sendTransaction(serializedTransaction);
-
   // We now need to impact/update our affected inputs
   // so we clear them out from UTXOset.
   impactAffectedInputs.call(this, {
-    inputs,
+    transaction,
+    txid,
   });
 
   return txid;
 }
 
+/**
+ * Broadcast a Transaction to the transport layer
+ * @param {Transaction|RawTransaction} transaction - A txobject or it's hexadecimal representation
+ * @param {Object} [options]
+ * @param {Boolean} [options.skipFeeValidation=false] - Allow to skip fee validation
+ * @return {Promise<transactionId>}
+ */
+async function broadcastTransaction(transaction, options = {}) {
+  if (!this.txISLockListener) {
+    const txId = await doBroadcastTransaction.call(this, transaction, options);
```

Comment:
Maybe something like `sendTransaction`?

Reply:
I would like to preserve the original meaning of the function to not introduce much confusion (like what is the difference between broadcast and send).

But I don't really like `do` prefix either. Let's make it `_broadcastTransaction` to indicate that `broadcastTransaction` is a wrapper for the actual logic?

Reply:
Does liner allow such underscored names?

Reply:
It doesn't. 
That's was the reason I used `do` prefix instead of the underscore. The rule can be suppressed for this particular case though.

Do you know whether this rule has been introduced deliberately? Or is it some default setting of the config?